code/game/machinery/doors/poddoor.dm

```diff
@@ -14,15 +20,16 @@
 	armor = list(MELEE = 50, BULLET = 100, LASER = 100, ENERGY = 100, BOMB = 50, BIO = 100, RAD = 100, FIRE = 100, ACID = 70)
 	resistance_flags = FIRE_PROOF
 	damage_deflection = 70
-	poddoor = TRUE
-	var/ertblast = FALSE //If this is true the blast door cannot be deconstructed
-	var/deconstruction = INTACT //For the deconstruction steps
+	var/datum/crafting_recipe/recipe
```

Comment:
No need for this variable to be in this scope, just look it up when deconstructing.